CMakeLists.txt

```diff
@@ -20,15 +20,16 @@ include(GNUInstallDirs)
 set(CMAKE_AUTOMOC ON)
 find_package(Qt5Widgets REQUIRED)
 find_package(Qt5XmlPatterns REQUIRED)
+find_package(OPENSCAP REQUIRED)
 
 add_definitions(-DQT_DISABLE_DEPRECATED_BEFORE=0)
 
 # This conditional is here to avoid checking openscap version if user supplies
 # custom openscap parameters.
-if (NOT DEFINED OPENSCAP_LIBRARIES OR NOT DEFINED OPENSCAP_INCLUDE_DIRS)
-    include("${CMAKE_ROOT}/Modules/FindPkgConfig.cmake")
-    pkg_check_modules(OPENSCAP REQUIRED libopenscap>=1.2.0)
```

Comment:
Does not seem to be applicable in the Windows World - probably best to maybe wrap if (WIN32) clause as to not break compat with Linux